### PR TITLE
Event batching

### PR DESCRIPTION
**Pull request by NikTheNak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ Then you have to set it inside the settings file located at `Aptabase/Reosources
 
 Based on the key, your `Host` will be selected. In the case of self-hosted versions a new `SelfHostURL` field will appear for input.
 
-App Version is automatically detected, but you will need to provide a `BuildNumber` as it may vary across different platforms. This allows you to specify a platform-specific build number to ensure accurate version tracking and compatibility.
+App Version is automatically detected, but you can override it with the `AppVersion` field. You will need to provide a `AppBuildNumber` as it may vary across different platforms. This allows you to specify a platform-specific build number to ensure accurate version tracking and compatibility.
```
Comment (goenning) [inline README.md]: As AppBuilderNumber is optional, should we have something like this instead?
```
You may want to provide a `AppBuildNumber` as ...
```

File: Runtime/Dispatcher.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+using System.Threading.Tasks;
+using AptabaseSDK.TinyJson;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace AptabaseSDK
+{
+    public class Dispatcher
+    {
+        private const string EVENT_ENDPOINT = "/api/v0/event";
```
Comment (goenning) [inline Runtime/Dispatcher.cs]: I assume we no longer need this?

File: Runtime/Environment.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                    break;
+                case RuntimePlatform.IPhonePlayer:
+                    var model = SystemInfo.deviceModel.ToLower();
+                    operatingSystem.osName = model.Contains("ipad") ? "iPadOS" : "iOS";
+                    break;
+                case RuntimePlatform.LinuxPlayer:
+                    operatingSystem.osName = "Linux";
+                    break;
+                case RuntimePlatform.OSXPlayer:
+                    operatingSystem.osName = "maxOS";
```
Comment (goenning) [inline Runtime/Environment.cs]: macOS :)

**Review by goenning — CHANGES_REQUESTED** [review]
looks good to me, just a few minor comments.

Is is common to copy thirdparty dependencies in Unity? We can't use nuget?

File: Runtime/Dispatcher.cs (hunk 2 above)
Comment (NikTheNak) [reply Runtime/Dispatcher.cs]: I left in for single events, but yeah not needed. Will remove

File: Runtime/Environment.cs (hunk 3 above)
Comment (NikTheNak) [reply Runtime/Environment.cs]: oops! updating

**Comment by NikTheNak** [comment]
> Is is common to copy thirdparty dependencies in Unity? We can't use nuget?
From all the packages I use yes. Have to do some workarounds to get it to work out of the box. We could load in Newtonsoft via package manager, but that could lead to other issues.

**Comment by goenning** [comment]
Thanks! Going to tag this as 0.2.0 🎉

**Pull request merged**